eval/src/vespa/eval/tensor/dense/onnx_wrapper.h

```diff
@@ -83,8 +84,9 @@ class Onnx {
     private:
         std::map<vespalib::string,eval::ValueType> _input_types;
         std::map<vespalib::string,size_t> _symbolic_sizes;
+        std::set<size_t> _unknown_sizes;
```

Comment:
consider renaming to "_resolved_sizes" since they are no longer unknown when added to the set...